src/GrpcProtos/capture.proto

```diff
@@ -349,9 +349,11 @@ message SystemMemoryUsage {
 message ProcessMemoryUsage {
   int32 pid = 1;
   uint64 timestamp_ns = 2;
+  // This field is retrieved from /proc/<pid>/status. See
+  // https://man7.org/linux/man-pages/man5/proc.5.html for the documentation.
+  int64 rss_anon_kb = 3;
   // These fields are retrieved from /proc/<pid>/stat. See
   // https://man7.org/linux/man-pages/man5/proc.5.html for the documentation.
-  int64 rss_pages = 3;
```

Comment:
That field was already used (in a released version), right? 
For backwards-compatibility, we should mark the field number 3 as `reserved` ([see here](https://developers.google.com/protocol-buffers/docs/proto#reserved)).

For the `rss_pages` you should then use a new number.

Reply:
The new field also has the id. So it's a rename only. No breaking change, right?

Reply:
Good point. I am not 100% if it is fine in that case, but I guess you are right.

Reply:
I guess the more important question is: What happens when a user opens a 1.65 capture with 1.66. We will read the capture and interpet the RSS value as AnonRSS. Is that a problem? Is it better to make a breaking change? Is it worth to invest more and make it backwards compatible while not showing wrong things?

Reply:
Thanks for pointing it out! I didn't think about this 🙈. As we discussed offline, I changed to make the field number 3 reserved.